### DISCUSSION TITLE
Quote

### DISCUSSION THREAD

**Discussion by couet** [artem-ogre/CDT#155] [General]
Hi,
As I said in a previous discussion we are planning to use CDT in the ROOT package. What is the best way to cite you?
Cheers,
Olivier

**Comment by artem-ogre** [comment]
Dear Olivier,

Thank you, that’s great to hear.
Does this section of README answer your question?
https://github.com/artem-ogre/CDT#license

**Comment by couet** [comment]
Yes exactly!! Many thanks!Envoyé de mon iPhoneLe 2 nov. 2023 à 19:43, Artem Amirkhanov ***@***.***> a écrit :﻿
Dear Olivier,
Thank you, that’s great to hear.
Does this section of README answer your question?
https://github.com/artem-ogre/CDT#license
